alacritty/src/macos/proc.rs

```diff
@@ -0,0 +1,104 @@
+use std::ffi::OsStr;
+use std::io;
+use std::mem;
+use std::os::raw;
+use std::os::unix::ffi::OsStrExt;
+use std::path::PathBuf;
+use std::slice;
+use memchr::memchr;
+
+pub fn cwd(pid: raw::c_int) -> io::Result<PathBuf> {
+    let flavor = PROC_PIDVNODEPATHINFO as raw::c_int;
```

Comment:
Can you not just define `PROC_PIDVNODEPATHINFO` as a c_int?

Reply:
Sure could! I started with the output of `bindgen` and didn't catch all the transforms as you can tell :wink:

Reply:
So stop casting it then?

Reply:
:sweat_smile: